doc/src/modules/index.rst

```diff
@@ -11,7 +11,7 @@ Most of the things are already documented though in this document, that is
 automatically generated using SymPy's docstrings.
 
 Click the  "modules" (:ref:`modindex`) link in the top right corner to easily
-access any SymPy module, or use this contens:
+access any SymPy module, or use this contents:
```

Comment:
Yeah, dictionaries say that "contents" is plural, but somehow "these contents" sounds awkward to me. (I'm not a native speaker of English.)
How about making it "this table of contents"?